packages/trader/src/Modules/Trading/Components/Form/form-layout.jsx

```diff
@@ -15,11 +16,15 @@ const Screen = Loadable({
     },
 });
 
-const FormLayout = ({ is_market_closed, is_trade_enabled }) => (
-    <React.Fragment>
-        <Screen is_trade_enabled={is_trade_enabled} is_market_closed={isMobile() ? undefined : is_market_closed} />
-    </React.Fragment>
-);
+const FormLayout = observer(({ is_market_closed, is_trade_enabled }) => {
+    const { common } = useStore();
```

Comment:
Can you do a TS migration on the file as well.

Reply:
ok will do.